packages/wdio-browserstack-service/src/launcher.ts

```diff
@@ -657,4 +700,15 @@ export default class BrowserstackLauncherService implements Services.ServiceInst
         jsonContent[buildName] = { 'identifier': buildIdentifier }
         fs.writeFileSync(filePath, JSON.stringify(jsonContent))
     }
+
+    _getClientBuildUuid() {
+        if (process.env.BS_TESTOPS_BUILD_HASHED_ID) {
+            return process.env.BS_TESTOPS_BUILD_HASHED_ID
+        } else if (process.env.BS_A11Y_TEST_RUN_ID) {
+            return process.env.BS_A11Y_TEST_RUN_ID
+        }
+        const uuid = uuidv4()
+        BStackLogger.debug(`If facing any issues, please contact BrowserStack support with the test run id - ${uuid}`)
```

Comment:
This log will be visible to all wdio users who are using debug logs. Are we aligned with product on showing this log?